### PR TITLE
Add Polish Identity Card number (dowód osobisty)

### PR DESCRIPTION
**Pull request by regdos**

### CODE REVIEW

File: docs/IdentityCard.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# IdentityCard
+
+- `v::identityCard()`
```
Comment (henriquemoody) [inline docs/IdentityCard.md]: I think this name is too generic to cover only Polish Identify Card, do you have a better name in mind?

Comment (regdos) [inline docs/IdentityCard.md]: Maybe a good idea is to move PESEL and Identity Card validators to Locale?

Comment (regdos) [inline docs/IdentityCard.md]: I add proof of concept commit (tests not working). 
Move PESEL and Identity Card to common Poland validator. Do you think that is that OK?

Comment (henriquemoody) [inline docs/IdentityCard.md]: I think we can keep it here for a while.

Even if it moves to Locale, the same is too generic as well, an
"IdentityCard" rule must be able to validate any identify card, or at least
more than one.

On Wed, Dec 23, 2015 at 6:19 PM, Tomasz Regdos <notifications@github.com>
wrote:

> In docs/IdentityCard.md
> <https://github.com/Respect/Validation/pull/623#discussion_r48376486>:
>
> > @@ -0,0 +1,12 @@
> > +# IdentityCard
> > +
> > +- `v::identityCard()`
>
> I add proof of concept commit (tests not working).
> Move PESEL and Identity Card to common Poland validator. Do you think that
> is that OK?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Respect/Validation/pull/623/files#r48376486>.
>



-- 
*Henrique Moody*

*Web Developer | ZEND016279*
http://about.me/henriquemoody
*+55 11 98183-2855*

Comment (regdos) [inline docs/IdentityCard.md]: What do you suggest?

Comment (henriquemoody) [inline docs/IdentityCard.md]: *Pesel* may be kept, I think IdentityCard accepting a county code on constructor may be the best option for us.

**Comment by henriquemoody** [comment]
Merged by c2eece6785121a239488badf8f95d137e48b0d5b.

Thanks for contributing! :panda_face:

**Pull request closed without merging**